Dockerfile

```
@@ -5,7 +5,7 @@ ENV REVIEWDOG_VERSION v0.10.2
 SHELL ["/bin/ash", "-eo", "pipefail", "-c"]
 
 # hadolint ignore=DL3018
-RUN apk add --update --no-cache build-base git grep
+RUN apk add --update --no-cache build-base git grep pcre-tools
```

```suggestion
RUN apk add --update --no-cache build-base git pcre-tools
```
Could we remove `grep` from this line?

---

We sure can, it's not used anywhere else